Tools/clinic/clinic.py

```diff
@@ -3033,7 +3056,7 @@ def methoddef_flags(self) -> str | None:
             case FunctionKind.STATIC_METHOD:
                 flags.append('METH_STATIC')
             case _ as kind:
-                assert kind is FunctionKind.CALLABLE, f"unknown kind: {kind!r}"
+                assert kind in {FunctionKind.CALLABLE, FunctionKind.GETTER}, f"unknown kind: {kind!r}"
```

Comment:
The line is a bit long here :)

```suggestion
                acceptable_kinds = {FunctionKind.CALLABLE, FunctionKind.GETTER}
                assert kind in acceptable_kinds, f"unknown kind: {kind!r}"
```